pages/Synthetic-Diamond.md

```diff
@@ -0,0 +1,4 @@
+The Synthetic Diamond is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources) that takes the place of a Vanilla Minecraft diamond, meaning it can be used for all the same purposes.
```

Comment:
only in the workbench though

Reply:
It's not usable anywhere else except in Slimefun. Unless something changed and you can now use Slimefun Resources outside of Slimefun.